### ISSUE TITLE
Give bus routes a higher penalty

### ISSUE THREAD

**Issue by abhumbla** [bikehopper/graphhopper#159]
maybe it should be 2

**Comment by graue** [comment]
Why?

**Comment by abhumbla** [comment]
there's a bunch of cases I've seen where we prefer the bus over the train where I don't think we should. I want to see what increasing the bus penalty will do. if it produces bad results we don't have to make it permanent. but even just getting the code in a state where I can give different modes of transit different personalities will be useful

**Comment by graue** [comment]
Makes sense. I changed the name of the issue to reflect that it's an experiment and not necessarily something we agree to do.

> there's a bunch of cases I've seen where we prefer the bus over the train where I don't think we should.

I agree with this. In those cases I usually see the bike->bus->bike route taking significantly longer than the bike->train->bike route would. I suspect it's largely because there are high penalties for the roads immediately around the train station. Alternative approaches to fix could be:

- Reduce penalties across the board. As I mentioned, I think we're often not doing a great job estimating the level of safety/comfort a route offers, so we could try de-emphasizing the penalties to reflect that limited confidence.
- Reduce penalties when close to a train station
- Instead of letting a bike->bus->bike route trump a bike->train->bike route, return both
- Install guerrilla protected bike lanes around BART stations

FWIW, bus can be better because quicker to access/less schlepping. In some ways we currently undercount the cost to use trains because we don't model the schlepping, or the full amount of time required to carry a bike down to the platform on elevators.
